tests/distribution/sympy/test_generating_function.py

```diff
@@ -161,6 +161,7 @@ def test_expected_value_of(self):
         # Linearity breaks if intermediate results are negative.
         with pytest.raises(ValueError):
             gf.get_expected_value_of("n - 7 + 1")
+        pytest.xfail("known bug, hard to fix")
```

Comment:
I think a more precise description why the fail happens is more useful. That his bug is known, is basically already stated as you mark it as xfail.

Reply:
I changed this, I hope the new version conveys the most important information